src/reducers/monitors.js

```diff
@@ -0,0 +1,27 @@
+const UPDATE_MONITORS = 'scratch-gui/monitors/UPDATE_MONITORS';
+
+const initialState = {
+    monitors: []
+};
+
+const reducer = function (state, action) {
+    if (typeof state === 'undefined') state = initialState;
+    switch (action.type) {
+    case UPDATE_MONITORS:
+        return Object.assign({}, state, {monitors: action.monitors});
```

Comment:
You need the top-level state to be an object, but as far as I know, the slices can be anything you want.